src/games/hex_state.h

```diff
@@ -80,6 +80,8 @@ void Hex::State<SIZE, PIE>::ApplyAction(const _Action& action) {
   // find board move from action
   int i = action.GetY();
   int j = action.GetZ();
+  assert(i >= 0 && i < SIZE);  // N added
```

Comment:
eh no need to comment `// N added`, we can see you added it in the Git blame

Reply:
my preference would be to remove the comments and just use Git blame because I think it leaves the code cleaner. But if you find leaving the comments to be useful then I think having them is NBD, it's understandable to me